sql/driver/doc.go

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+Package driver provides a standard database/sql compatible SQL driver for Hazelcast.
+
+This driver supports Hazelcast 5.0 and up. Check out the Hazelcast SQL documentation here: https://docs.hazelcast.com/hazelcast/latest/sql/sql-overview
+
+The documentation for the database/sql package is here: https://pkg.go.dev/database/sql
+
+Enabling Hazelcast SQL
+
+The SQL support should be enabled in Hazelcast configuration:
```

Comment:
```suggestion
The SQL support should be enabled in Hazelcast member configuration:
```
I feel these configurations gets mixed up quite often from experiences on CLC. Maybe we can clarify that not the client config?

Reply:
I agree, updated at: https://github.com/hazelcast/hazelcast-go-client/pull/709/commits/f9a3272a3f7266e575f052f13da0ae342c3d20d8